ironfish/src/wallet/wallet.ts

```diff
@@ -1303,28 +1303,60 @@ export class Wallet {
     }
   }
 
-  async importAccount(toImport: AccountImport): Promise<Account> {
-    if (toImport.name && this.getAccountByName(toImport.name)) {
-      throw new Error(`Account already exists with the name ${toImport.name}`)
+  async importSpendingAccount(toImport: AccountImport): Promise<Account> {
```

Comment:
Now that we have two methods we should use the two types, that way we wouldn't need this assertion